test/libsolidity/ASTJSON/documentation_local_variable.sol

```diff
@@ -0,0 +1,25 @@
+contract C {
+    function f() public pure {
+        /// Documentation for x; will appear in ast json
+        uint x = 1;
```

Comment:
The parse only JSON would already contain "documentation for x, ..." even without this fix.